### ISSUE TITLE
print the version once if there's an exact match to semantic version

### ISSUE THREAD

**Issue by jasinner** [release-engineering/backvendor#33]
In the example output in the README there is this line:

github.com/opencontainers/image-spec@87998cd070d9e7a2c79f8b153a26bea0425582e5 =v1.0.0 ~v1.0.0

Why do we bother printing ~v1.0.0 does it have any meaning?

**Comment by jasinner** [comment]
Perhaps it's the =v1.0.0 part we should leave out? Or do you prefer it this way because it show's it not a generated semantic version tag?

**Comment by jasinner** [comment]
This is a nit-pick BTW.

**Comment by twaugh** [comment]
Yes, the pseudo-version field is not really needed if the exact-version field is available.

**Comment by jasinner** [comment]
Actually, it's more useful for our immediate needs if we print versions out like this:

github.com/opencontainers/image-spec:v1.0.0 (for exact match)

or

k8s.io/gengo:v0.0.0-0.20180529112933-461b8bcef4e1 (for generated version)

Do you think we could change the output format?

**Comment by twaugh** [comment]
So, `[*]$PROJECT:$VERSION`, same regardless of whether VERSION is an exact match or a pseudo-version? I think it's still useful to keep the asterisk marker to indicate the top-level project.

**Comment by jasinner** [comment]
Yes [ * ] $PROJECT:$VERSION
Agreed on keeping Asterix.

**Comment by twaugh** [comment]
Some users of this program will definitely still want to record the revision ID, even in the case of an exact tag match.

I was hoping we could make the format configurable using a template.
